plugin/bufmode.lua

```diff
@@ -1,6 +1,2 @@
-local bufmode = require 'bufmode'
-
-vim.keymap.set('n', '<Plug>(BufmodeEnter)', bufmode, {silent = true, unique = true})
-vim.keymap.set('n', '<leader>b', '<Plug>(BufmodeEnter)', {remap = true, silent = true, unique = true})
-
-vim.api.nvim_create_user_command('BufmodeEnter', bufmode, {force = false})
+local bufmode = require'bufmode'
+bufmode.setup()
```

Comment:
You can just do `require'bufmode'.setup()` or `require('bufmode').setup()` since `bufmode` is only referenced once (I personally prefer the latter in this case)

Reply:
done